connection.go

```diff
@@ -118,26 +145,26 @@ func (c *conn) runQuery(ctx context.Context, query string, args []driver.NamedVa
 		switch opStatus.GetOperationState() {
 		// terminal states
 		// good
-		case ts.TOperationState_FINISHED_STATE:
+		case cli_service.TOperationState_FINISHED_STATE:
 			// return results
 			return exStmtResp, opStatus, nil
 		// bad
-		case ts.TOperationState_CANCELED_STATE, ts.TOperationState_CLOSED_STATE, ts.TOperationState_ERROR_STATE, ts.TOperationState_TIMEDOUT_STATE:
+		case cli_service.TOperationState_CANCELED_STATE, cli_service.TOperationState_CLOSED_STATE, cli_service.TOperationState_ERROR_STATE, cli_service.TOperationState_TIMEDOUT_STATE:
 			// do we need to close the operation in these cases?
 			log.Debug().Msgf("bad state: %s", opStatus.GetOperationState())
 			log.Error().Msg(opStatus.GetErrorMessage())
 
 			return exStmtResp, opStatus, fmt.Errorf(opStatus.GetDisplayMessage())
 		// live states
-		case ts.TOperationState_INITIALIZED_STATE, ts.TOperationState_PENDING_STATE, ts.TOperationState_RUNNING_STATE:
+		case cli_service.TOperationState_INITIALIZED_STATE, cli_service.TOperationState_PENDING_STATE, cli_service.TOperationState_RUNNING_STATE:
 			statusResp, err := c.pollOperation(ctx, opHandle)
 			if err != nil {
 				return nil, statusResp, err
 			}
 			switch statusResp.GetOperationState() {
 			// terminal states
 			// good
-			case ts.TOperationState_FINISHED_STATE:
+			case cli_service.TOperationState_FINISHED_STATE:
 				// return handle to fetch results later
 				return exStmtResp, opStatus, nil
 			// bad
```

Comment:
Nit: a lot of nesting of cases here, no functional problems but I'll pull this out

Reply:
yes. But it is still pretty readable, so I left the way it is to not complicate further. If it bothers you feel free to make the changes!